client/blocks/plan-thank-you-card/index.jsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import Gridicon from 'components/gridicon';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { fetchSitePlans } from 'state/sites/plans/actions';
+import { getPlanBySlug, getPlans } from 'state/plans/selectors';
+import { requestPlans } from 'state/plans/actions';
+
+class PlanThankYouCard extends Component {
+	static propTypes = {
+		selectedSite: PropTypes.object
+	};
+
+	componentDidMount() {
```

Comment:
We prefer to use data query components instead of calling the actions manually. See https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plans/current-plan/index.jsx#L82-L82 for how to do it. :)